catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandlerTest.java

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.handlers;
+
+import static junit.framework.TestCase.assertNull;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import ddf.catalog.data.BinaryContent;
+import ddf.catalog.data.impl.BinaryContentImpl;
+import ddf.catalog.data.types.Core;
+import ddf.catalog.operation.QueryResponse;
+import ddf.catalog.transform.QueryResponseTransformer;
+import java.io.ByteArrayInputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.activation.MimeType;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.HttpHeaders;
+import org.codice.ddf.catalog.ui.query.cql.CqlQueryResponse;
+import org.codice.ddf.catalog.ui.query.cql.CqlRequest;
+import org.codice.ddf.catalog.ui.util.EndpointUtil;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceReference;
+import spark.Request;
+import spark.Response;
+
+public class CqlTransformHandlerTest {
+
+  private List<ServiceReference> queryResponseTransformers;
+  private CqlTransformHandler cqlTransformHandler;
+  private BinaryContent binaryContent;
+
+  @Mock private ServiceReference<QueryResponseTransformer> mockQueryResponseTransformer;
```

Comment:
Do these tests actually run with @mock annotation without specifying the mockito test runner?

Reply:
they do

Reply:
@Schachte You can use the `RunWith` test runner or you can use `initMocks(this)` in your setup method.